zcash-issue-dag.py

```diff
@@ -245,88 +267,168 @@ def chunks(lst, n):
 
     return ret
 
+def fetch_workspace_graph(op, workspace_id, repos, cursor):
+    dependencies = op.workspace(id=workspace_id).issue_dependencies(
+        # TODO: This causes a 500 Internal Server Error. We need the ZenHub repo IDs here,
+        # not the GitHub repo IDs (which the previous REST API used).
+        # repository_ids=repos,
+        first=100,
+        after=cursor,
+    )
+    dependencies.nodes.id()
```

Comment:
These calls don't appear to do anything other than check that the relevant methods exist. Is that the point of calling them here?